lib/parsers/project-assets-json-parser.ts

```diff
@@ -0,0 +1,78 @@
+import * as _ from 'lodash';
+
+export interface PkgTree {
+  name: string;
+  version: string;
+  dependencies: {
+    [dep: string]: PkgTree;
+  };
+  targetFrameworks?: string[];
+}
+
+export interface ProjectAssetsJsonManifest {
+  targets: {
+    [target: string]: {
+      [name: string]: {
+        type: string,
+        dependencies?: {
+          [deps: string]: string;
+        };
+      };
+    },
+  };
+  project: {
+    restore: {
+      projectName: string,
+    },
+    version: string,
+  };
+}
+
+export function getProjectNameForProjectAssetsJson(manifestFile) {
+  return _.get(manifestFile, ['project', 'restore', 'projectName'], {});
```

Comment:
Im not sure how to use a variable with _get and dot notation .e.g like for `topLevelDeps = Object.keys(_.get(manifestFile, ['targets', targetFrameWork], {}));` where targetFrameworks is passed in. 
If you can provide an example, I am happy to change it.

Reply:
In case you mentioned, the only way would be:
```
topLevelDeps = Object.keys(_.get(manifestFile, `targets.${targetFrameWork}`, {}));
```

But I don't think it's more readable :-) My original comment was for the case without variable.

Reply:
Cool, I can update the variables without variables with dot notation if that is more readable :)

Reply:
Tried updating to use dotnotation but was not able to retrieve the project name and version :(